### DISCUSSION TITLE
Impossibile aggiornare HACS

### DISCUSSION THREAD

**Discussion by GiulioFattoretto** [indomus/forum#1728] [HUB: Home Assistant]
Buongiorno, scrivo in quanto sono diversi giorni che provo a risolvere questo problema ma non ci sono riuscito.
HACS non si aggiorna più, non riesco ad aggiornare HACS (fermo alla versione 1.13.2, ora 1.25.5 disponibile) e cosi di seguito anche tutte le integrazioni che avevo installato non si riescono ad aggiornare, ovviamente neanche nuove integrazioni possono essere scaricate ed installate. Qualcuno riuscirebbe a darmi una mano?
Qui sotto le schermate del sistema attualmente installato e configurato.
![Schermata 2022-06-11 alle 14 31 45](https://user-images.githubusercontent.com/78276738/173188075-0ba19be1-82e1-401f-8136-1d80620b7904.png)
![Schermata 2022-06-11 alle 14 31 59](https://user-images.githubusercontent.com/78276738/173188089-2a97ce4e-d9eb-4e2b-af49-20e7f29c8433.png)
![Schermata 2022-06-11 alle 14 32 16](https://user-images.githubusercontent.com/78276738/173188094-5b85eec0-a445-4fed-ba3e-3e65be1be983.png)

**Comment by snakuzzo** [comment]
In base a quello che hai scritto è IMPOSSIBILE darti una mano.
Devi dettagliare bene con evidenze (esempio...erroir, messaggi di log, etc).
